ui-speedspacechart/src/components/const.ts

```diff
@@ -4,6 +4,13 @@ export const SLOPE_FILL_COLOR = '#CFDDCE';
 
 export const RIGHT_TICK_HEIGHT_OFFSET = 2;
 
+export const TICK_ICONS = {
```

Comment:
I think `TICK_TITLE_MARGINS` would be a better name, what do you think ?
```suggestion
export const TICK_TITLE_MARGINS = {
```

Reply:
You're right, done 👍